WinUIGallery/App.xaml.cs

```diff
@@ -42,15 +36,20 @@ public static string WinAppSdkDetails
             // TODO: restore patch number and version tag when WinAppSDK supports them both
             get => string.Format("Windows App SDK {0}.{1}",
                 WASDK.Release.Major, WASDK.Release.Minor);
-        }
-
+        }
+
         public static string WinAppSdkRuntimeDetails
         {
             get
-            {
-                var details = WinAppSdkDetails;
+            {
+                var details = WinAppSdkDetails;
 #if WindowsAppSdkRuntimeDependent
```

Comment:
>WindowsAppSdkRuntimeDependent [](http://example.com/codeflow?start=4&length=29)

With the new implementation, is there any need to condition this?

Reply:
well no, come to think of it